### DISCUSSION TITLE
Key match for search

### DISCUSSION THREAD

**Discussion by Tychodewaard** [dnnsoftware/Dnn.Platform#3128] [Ideas]
## Description of problem
If there are several pages with a certain keyword, the best page is not always the number 1 position in the search results.

## Description of solution
I have several customers who would like to have a 'key match' in the search results. This means that a certain keyword has a results page with a defined number 1. So, I can control that if someone who puts in 'bread' in the search box, that the page /great-bread will be on the number one position in the results list. 
Preferrably with the ability to have a different look&feel.

For the back end we would need:
- a subtab within Settings - Site settings - Search 
- ability to add a keyword and a dropdown to select a page
if multiple keywords are related to the same page, we can use the synonyms tab. 

To keep it simple: 
I would suggest that the key word match does not need to be linked to fuzzy search. If it works rightaway, perfect but not a must have.

**Comment by stale** [comment]
We have detected this issue has not had any activity during the last 90  days. That could mean this issue is no longer relevant and/or nobody has  found the necessary time to address the issue. We are trying to keep the  list of open issues limited to those issues that are relevant to the  majority and to close the ones that have become 'stale' (inactive). If no  further activity is detected within the next 14 days, the issue will be  closed automatically.
If new comments are are posted and/or a solution (pull request) is  submitted for review that references this issue, the issue will not be  closed. Closed issues can be reopened at any time in the future. Please remember those participating in this open source project are volunteers  trying to help others and creating a better DNN Platform for all. Thank  you for your continued involvement and contributions!

**Comment by Tychodewaard** [comment]
Reply to reactivate
________________________________
Van: stale[bot] <notifications@github.com>
Verzonden: maandag 30 maart 2020 09:25
Aan: dnnsoftware/Dnn.Platform <Dnn.Platform@noreply.github.com>
CC: Tycho de Waard <info@tjeps.com>; Author <author@noreply.github.com>
Onderwerp: Re: [dnnsoftware/Dnn.Platform] Key match for search (#3128)


We have detected this issue has not had any activity during the last 90 days. That could mean this issue is no longer relevant and/or nobody has found the necessary time to address the issue. We are trying to keep the list of open issues limited to those issues that are relevant to the majority and to close the ones that have become 'stale' (inactive). If no further activity is detected within the next 14 days, the issue will be closed automatically.
If new comments are are posted and/or a solution (pull request) is submitted for review that references this issue, the issue will not be closed. Closed issues can be reopened at any time in the future. Please remember those participating in this open source project are volunteers trying to help others and creating a better DNN Platform for all. Thank you for your continued involvement and contributions!

**Comment by valadas** [comment]
Is there anyone that wan't to develop this or should this be moved into a discussion until someone shows interest in implementing it ?

**Comment by Tychodewaard** [comment]
Is discussion the place where we discuss the desired way of implementing something or is it the 'ideas' corner where we upvote ideas in order to prioritize? Or both. 

As long as it is not the 'stale corner' which would be the equivalent of 'ignore corner' :-)

**Comment by valadas** [comment]
Well one thing that everybody needs to understand is that this is an open source project and contributors are not paid. Ideas are good but it clutters the backlog if nobody picks them up and just revive stale issues.

We implemented stalebot to help us clean that backlog by closing inactive issues, now if the issue is a new feature and people just comment to revive them without any hopes of someone actually writing that code, it kinda defeats the purpose of stalebot.

It's not that anybody likes to ignore those but it is just cleaning up the backlog to keep actionable issue that someone will actually implement any soon. Discussions are enabled now and have an upvote feature to them, this is something we can use now. For the ideas module, yes it may be better or it may also just duplicate what the github discussions does, but it is not yet implemented.

That is why I try to comment on reopen stale issue to ask if anybody has the intention to implement the thing. So if nobody replies again for another 90, we can kinda assume nobody has the intetion to implement it.

**Comment by stale** [comment]
We have detected this issue has not had any activity during the last 90  days. That could mean this issue is no longer relevant and/or nobody has  found the necessary time to address the issue. We are trying to keep the  list of open issues limited to those issues that are relevant to the  majority and to close the ones that have become 'stale' (inactive). If no  further activity is detected within the next 14 days, the issue will be  closed automatically.
If new comments are are posted and/or a solution (pull request) is  submitted for review that references this issue, the issue will not be  closed. Closed issues can be reopened at any time in the future. Please remember those participating in this open source project are volunteers  trying to help others and creating a better DNN Platform for all. Thank  you for your continued involvement and contributions!

**Comment by Tychodewaard** [comment]
Reply

Outlook voor iOS<https://aka.ms/o0ukef> downloaden
________________________________
Van: stale[bot] <notifications@github.com>
Verzonden: Thursday, July 2, 2020 12:43:06 AM
Aan: dnnsoftware/Dnn.Platform <Dnn.Platform@noreply.github.com>
CC: Tycho de Waard <info@tjeps.com>; Author <author@noreply.github.com>
Onderwerp: Re: [dnnsoftware/Dnn.Platform] Key match for search (#3128)


We have detected this issue has not had any activity during the last 90 days. That could mean this issue is no longer relevant and/or nobody has found the necessary time to address the issue. We are trying to keep the list of open issues limited to those issues that are relevant to the majority and to close the ones that have become 'stale' (inactive). If no further activity is detected within the next 14 days, the issue will be closed automatically.
If new comments are are posted and/or a solution (pull request) is submitted for review that references this issue, the issue will not be closed. Closed issues can be reopened at any time in the future. Please remember those participating in this open source project are volunteers trying to help others and creating a better DNN Platform for all. Thank you for your continued involvement and contributions!

**Comment by stale** [comment]
We have detected this issue has not had any activity during the last 90  days. That could mean this issue is no longer relevant and/or nobody has  found the necessary time to address the issue. We are trying to keep the  list of open issues limited to those issues that are relevant to the  majority and to close the ones that have become 'stale' (inactive). If no  further activity is detected within the next 14 days, the issue will be  closed automatically.
If new comments are are posted and/or a solution (pull request) is  submitted for review that references this issue, the issue will not be  closed. Closed issues can be reopened at any time in the future. Please remember those participating in this open source project are volunteers  trying to help others and creating a better DNN Platform for all. Thank  you for your continued involvement and contributions!

**Comment by Tychodewaard** [comment]
Unstale

Outlook voor iOS<https://aka.ms/o0ukef> downloaden
________________________________
Van: stale[bot] <notifications@github.com>
Verzonden: Tuesday, October 27, 2020 12:58:00 AM
Aan: dnnsoftware/Dnn.Platform <Dnn.Platform@noreply.github.com>
CC: Tycho de Waard <info@tjeps.com>; Author <author@noreply.github.com>
Onderwerp: Re: [dnnsoftware/Dnn.Platform] Key match for search (#3128)


We have detected this issue has not had any activity during the last 90 days. That could mean this issue is no longer relevant and/or nobody has found the necessary time to address the issue. We are trying to keep the list of open issues limited to those issues that are relevant to the majority and to close the ones that have become 'stale' (inactive). If no further activity is detected within the next 14 days, the issue will be closed automatically.
If new comments are are posted and/or a solution (pull request) is submitted for review that references this issue, the issue will not be closed. Closed issues can be reopened at any time in the future. Please remember those participating in this open source project are volunteers trying to help others and creating a better DNN Platform for all. Thank you for your continued involvement and contributions!

**Comment by valadas** [comment]
As per previous discussion, I am moving this to a discussion since nobody showed interest in implementing this yet.
